src/coreos-assembler

```diff
@@ -74,14 +74,6 @@ if [ -z "${cmd}" ]; then
 fi
 shift
 
-COSA_META_SCHEMA="${COSA_META_SCHEMA:-/usr/lib/coreos-assembler/v1.json}"
-schema_override="${PWD}/src/config/schema.json"
-if [ -e "${schema_override}" ]; then
-    COSA_META_SCHEMA=$(realpath "${schema_override}")
-fi
-export COSA_META_SCHEMA
```

Comment:
so the code itself already behaves this way and there was no reason to do this?

Does it handle if `"${PWD}/src/config/schema.json"` exists?

I'm happy to delete this code, just don't know if anything is relying on that.

Reply:
> Does it handle if "${PWD}/src/config/schema.json" exists?

Ah right, I should have commented on that.  I don't believe that was ever actually used, so it should be safe to delete.